woocommerce/api/class-sv-wc-api-base.php

```diff
@@ -548,6 +571,79 @@ protected function get_request_duration() {
 	}
 
 
+	/**
+	 * Gets the request transient key for the current plugin and request data.
+	 *
+	 * Request transients can be disabled by using the filter below.
+	 *
+	 * @since 5.10.10
+	 *
+	 * @return string transient key
+	 */
+	protected function get_request_transient_key() : string {
+
+		// ex: wc_sv_requests_<md5 hash of plugin_id, request uri and request data>
+		return sprintf( 'wc_sv_requests_%s', md5( implode( '_', [
```

Comment:
any reason these keys need to be generic to the FW? I know the md5 hash will make it unique to the plugin and request technically, but also obfuscates the db entry and makes for harder debugging. What about something like:

`wc_{plugin_id}_api_response_{hash}`? Also noting that the value we're caching here is essentially the response, not the request.

Reply:
I think having the plugin_id there makes perfect sense, thanks! And yeah, naming the key like this makes it clear that it's the response that's cached.